lf/demo/demo_robot.lf

```diff
@@ -2,24 +2,53 @@ target C {
     timeout: 15 sec
 };
 
-reactor Sensors {    
+// This is what we call a driver container. But I think at Luos we will have 2 drivers, one for the button and one for bumper.
+// But perhaps its more lik nodes.
+reactor Sensors {
+    // Those are like pipe beetween reactors?
```

Comment:
Kind of. You can think of ports as variables that are visible outside of the reactor. We can pipe data between 2 ports with the connection statement, [L217](https://github.com/nicolas-rabault/lf-flexpret-scripts/pull/1/commits/b8b0396a1e5824efb9cd02edefecdb89ac1c9462#diff-635897dda802e4648f1da919b6f4857a9040f3fe19f2fa1db5c96aa0583b3de6R217) for example.